releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,47 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.1/1.2.1 - January 13, 2020
+
+* Added Support for Spring Boot 2.2.x Hoxton with version 1.2.1. Spring Boot 2.1.x Greenwich is still supported with 1.1.1.
+
+### Configuration
+
+* 4 configuration parameters have renamed/replaced/removed. See [starter](https://github.com/microsoft/spring-cloud-azure/blob/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md) for updated descriptions.
+
+```properties
+# The following configurations have been removed
+spring.cloud.azure.appconfiguration.watch.enabled
+spring.cloud.azure.appconfiguration.managed-identity.object-id
+
+# The following property has been renamed
+spring.cloud.azure.appconfiguration.watch.delay -> spring.cloud.azure.appconfiguration.cache.expiration
```

Comment:
Should it be spring.cloud.azure.appconfiguration.**cache-expiration**?

Reply:
No, before it was an object called watch with a delay Duration being set, now it is just the renamed object to cache with exiration Duration being set.

Reply:
Yes